lib/config.js

```diff
@@ -170,7 +191,9 @@ Config.prototype.buildArgs = function () {
 Config.prototype.shouldSign = function () {
   // it doesn't make sense to sign debug builds because the restrictions on loading
   // dynamic libs prevents them from working anyway
-  return this.mac_signing_identifier !== '' && this.buildConfig === 'Release'
+  return this.mac_signing_identifier !== '' &&
```

Comment:
should `--skip_signing` and `mac_installer_signing_identifier` be checked here as well?

Reply:
this just skips the entire thing for ios no matter what so `--skip_signing` would be redundant. All I did here was add `this.targetOS !== 'ios'`